telemetry/definitions/commonDefinitions.json

```diff
@@ -1670,11 +1735,21 @@
             "type": "string",
             "description": "Date/time that an SSO client registration expires."
         },
+        {
+            "name": "step",
+            "type": "string",
+            "description": "Indicates the stage at which a user interface click action was performed." 
+        },
         {
             "name": "successCount",
             "type": "int",
             "description": "The number of successful operations"
         },
+        {
+            "name": "supportedLanguage",
+            "type": "boolean",
+            "description": "supported language"
+        },
```

Comment:
no.

Reply:
I didn't see a field which checks the supported language(boolean).
LMK If I am missing something.

Reply:
This field should be isSupportedLanguage to indicate it is boolean field. SupportedLanguage seems like we are going to pass the actual value of language which could be string/enum type.

Also the comment should be "Indicate if the language is supported"

Reply:
I did not see convention naming isXXXX for boolean values in the json file so for keeping consistent went with `supportedLanguage`
Let me change this accordingly